builder/amazon/common/regions.go

```diff
@@ -27,11 +33,12 @@ func listEC2Regions(ec2conn ec2iface.EC2API) []string {
 
 // ValidateRegion returns true if the supplied region is a valid AWS
 // region and false if it's not.
-func ValidateRegion(region string, ec2conn ec2iface.EC2API) bool {
-	for _, valid := range listEC2Regions(ec2conn) {
+func ValidateRegion(region string, ec2conn ec2iface.EC2API) error {
```

Comment:
I like this change. Much better.

Reply:
Yay 😄

Reply:
Ah, this makes me see I forgot to update func doc